fili-core/src/main/java/com/yahoo/bard/webservice/web/handlers/SplitQueryRequestHandler.java

```diff
@@ -63,6 +64,11 @@ public boolean handleRequest(
         Granularity granularity = druidQuery.getGranularity();
         List<Interval> queryIntervals = druidQuery.getIntervals();
 
+        if (granularity instanceof AllGranularity) {
+           // by-pass splitting for "all" grain
```

Comment:
It would be good to include _why_ we can't split for "all" grain. (ie. because there's only 1 response bucket, and so we can't split it)